### ISSUE TITLE
Can pretrained/finetuned TOD BERT be able to check when not to predict response in case of response selection task?

### ISSUE THREAD

**Issue by mayankgoyal1993** [jasonwu0731/ToD-BERT#21]
Can pretrained/finetuned TOD BERT be able to check when not to predict response in case of response selection task?

Since it works on softmax loss like classification settings, what needs to be done to be able to make sure it does not select either of the response for some out of domain conversation happening on bot?

**Comment by jasonwu0731** [comment]
Hello,

In order to know when not to respond, this is related to out-of-distribution detection and model calibration. The easiest way. could be fine-tuned TOD-BERT using binary cross entropy loss and calibrated it to the dev set probability. And find a fixed or dynamic threshold.

**Comment by mayankgoyal1993** [comment]
Thanks for the response, let me try to do that.
